oneview/resource_logical_interconnect_group.go

```diff
@@ -59,7 +59,7 @@ func resourceLogicalInterconnectGroup() *schema.Resource {
 			},
 			"interconnect_map_entry_template": {
 				Optional: true,
```

Comment:
This field is a mandatory field. So make it as required instead of optional.

Reply:
This is fixed now